hardware_interface/src/robot_hardware.cpp

```diff
@@ -25,18 +25,27 @@
 namespace hardware_interface
 {
 
+constexpr char kJointStateLoggerName[] = "joint state handle";
+constexpr char kJointCommandLoggerName[] = "joint cmd handle";
+constexpr char kOperationModeLoggerName[] = "joint operation mode handle";
+
 template<typename T>
 hardware_interface_ret_t
-register_handle(std::vector<T *> & registered_handles, T * handle)
+register_handle(
+  std::vector<T *> & registered_handles, T * handle,
+  const std::string & logger_name)
```

Comment:
nit: the argument here could fit on one line. Either line break after each argument or no line break at all.

Reply:
Fixed. https://github.com/ros-controls/ros2_control/pull/78/commits/3e9744cac845d24d5ad42f43e20fcc3011316a93